README.md

```diff
@@ -193,39 +213,40 @@ The tool runs a series of analyzers to ensure submitted plugins are following be
-| Checksum / `checksum` | Validates that the passed checksum (as a validator arg) is the one calculated from the archive file. | `checksum` | 
+| Broken Links / `brokenlinks`                      | Detects if any URL doesn't resolve to a valid location.                                                                           | None                                                                  |
+| Checksum / `checksum`                             | Validates that the passed checksum (as a validator arg) is the one calculated from the archive file.                              | `checksum`                                                            |
 | Code Rules / `coderules`                          | Checks for forbidden access to environment variables, file system or use of `syscall` module.                                     | [semgrep](https://github.com/returntocorp/semgrep), `sourceCodeUri`   |
-| Discoverability / `discoverability` | Warns about missing keywords and description that are used for plugin indexing in the catalog. | None |
+| Discoverability / `discoverability`               | Warns about missing keywords and description that are used for plugin indexing in the catalog.                                    | None                                                                  |
 | Go Manifest / `gomanifest`                        | Validates the build manifest.                                                                                                     | `sourceCodeUri`                                                       |
 | Go Security Checker / `gosec`                     | Inspects source code for security problems by scanning the Go AST.                                                                | [gosec](https://github.com/securego/gosec), `sourceCodeUri`           |
-| Developer Jargon / `jargon`                       | Generally discourages use of code jargon in the documentation.                                                                     | None                                                                |
-| JS Source Map / `jssourcemap`                     | Checks for required `module.js.map` file(s) in archive.                                                                             | `sourceCodeUri`                                                       |
-| Legacy Grafana Toolkit usage / `legacybuilder` | Detects the usage of the not longer supported Grafana Toolkit. | None |
+| Nested includes metadata / `includesnested`       | Validates that nested plugins have the correct metadata.                                                                          |                                                                       |
```

Comment:
This line is new. All the others modification are auto-formatting shifting because of the longer content